utils/s2n_safety_macros.h

```diff
@@ -38,91 +38,95 @@
 /**
  * Sets the global `s2n_errno` to `error` and returns with an `S2N_RESULT_ERROR`
  */
-#define RESULT_BAIL(error)                                     do { _S2N_ERROR((error)); return S2N_RESULT_ERROR; } while (0)
+#define RESULT_BAIL(error)       \
```

Comment:
This header is labeled "DO NOT DIRECTLY MODIFY THIS FILE" :)

If you want to format it, you'll need to change what the script that generates it (scripts/s2n_safety_macros.py) produces. I'd suggest just adding "clang-format off" to the script.

Reply:
Missed that is the review. Must have only looked at the diff. :)